bundles/org.openhab.binding.radiothermostat/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="radiothermostat" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- RadioThemostat Thing -->
+	<thing-type id="rtherm">
+		<label>Thermostat</label>
+		<description>
+			A Thermostat to control the house's HVAC system
+		</description>
+
+		<channels>
+			<channel id="name" typeId="name"/>
+			<channel id="model" typeId="model" />
+			<channel id="temperature" typeId="temp-temperature" />
+			<channel id="humidity" typeId="humidity" />
+			<channel id="mode" typeId="mode" />
+			<channel id="fan_mode" typeId="fan_mode" />
+			<channel id="program_mode" typeId="program_mode" />
+			<channel id="set_point" typeId="temp-sp" />
+			<channel id="override" typeId="override" />
+			<channel id="hold" typeId="hold" />
+			<channel id="status" typeId="status" />
+			<channel id="fan_status" typeId="fan_status" />
+			<channel id="day" typeId="t_day" />
+			<channel id="hour" typeId="t_hour" />
+			<channel id="minute" typeId="t_minute" />
+			<channel id="dt_stamp" typeId="dt_stamp" />
+			<channel id="last_update" typeId="last_update" />
+			<channel id="today_heat_runtime" typeId="today_heat_runtime" />
+			<channel id="today_cool_runtime" typeId="today_cool_runtime" />
+			<channel id="yesterday_heat_runtime" typeId="yesterday_heat_runtime" />
+			<channel id="yesterday_cool_runtime" typeId="yesterday_cool_runtime" />
+			<channel id="json_cmd" typeId="json_cmd" />
+		</channels>
+		
+		<config-description>
+			<parameter name="hostName" type="text" required="true">
+				<context>network-address</context>
+				<label>Thermostat Host Name/IP Address</label>
+				<description>Host name or IP address of the thermostat</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="1" required="false">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in minutes</description>
+				<default>2</default>
+			</parameter>
+			<parameter name="logRefresh" type="integer" min="5" required="false">
+				<label>Run-time Log Refresh Interval</label>
+				<description>Specifies the run-time log and humidity refresh interval in minutes</description>
+				<default>10</default>
+			</parameter>
+			<parameter name="disableLogs" type="integer" min="0" max="1" required="false">
+				<label>Disable retrieval of run-time data</label>
```

Comment:
The words in labels should be capitalized: https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Fixed!